test/TestUtils.py

```diff
@@ -154,3 +155,15 @@ def test_normpath_with_string(self):
         self.assertEqual(
             utils.normpath("a/b/../"),
             "a")
+
+    def test_expand_path_vars(self):
+        test_path = '/test/path'
+        os.environ['TEST_PATH'] = test_path
+        self.assertEqual(utils.expand_path_vars('~'), os.path.expanduser('~'))
```

Comment:
And also use `assert` statements to do checks as we now use `pytest` as a test runner:
```suggestion
    assert utils.expand_path_vars('~') == os.path.expanduser('~')
```

Reply:
I was following the paradigm that was used in the file. I will change them